modules/administration-guide/partials/proc_building-a-custom-plug-in-registry-image.adoc

```diff
@@ -19,9 +19,6 @@ The image contains plug-ins or extensions metadata.
 * `./node_modules/.bin` is in the `PATH` environment variable.
 
 * A running installation of link:http://podman.io[podman] or link:http://docker.io[docker].
-ifeval::["{project-context}" != "che"]
-* Valid content for the `meta.yaml` file describing the plug-in to add. See: xref:end-user-guide:publishing-metadata-for-a-vs-code-extension.adoc[]
-endif::[]
```

Comment:
should we keep it ?

Reply:
Yes it's still used in docs that describe the manual creation of `meta.yaml` files, for example when you want to use a custom plugin definition in a devfile via gist.